integration_tests/features/support/steps.js

```diff
@@ -3826,15 +3826,15 @@ Then(
       "Waiting for " + wallet_name + " to receive " + amount + " transaction(s)"
     );
 
-    await waitFor(
-      async () => {
-        return wallet.getCounters().received >= amount;
-      },
-      true,
-      700 * 1000,
-      5 * 1000,
-      5
-    );
+    let count = 0;
+
+    while (!(wallet.getCounters().received >= amount)) {
+      await sleep(1000);
```

Comment:
Why is this better?

Reply:
Wrote the one like that and updated the others so they were all all running the same code, that's the only reason.

Would you like me to change it back to waitFor for all of them?

Reply:
yes, unless there is a reason to not use waitFor?